src/Foundatio.Core/Utility/TaskHelper.cs

```diff
@@ -3,7 +3,7 @@
 using System.Threading.Tasks;
 
 namespace Foundatio.Utility {
-    internal static class TaskHelper {
+    public static class TaskHelper {
```

Comment:
Looks like we can do a file link like this (example): “buildOptions:” { “include”: [“../Folder/**.*.cs”] }